### PR TITLE
Categories reorg

### PR DESCRIPTION
**Pull request by caldav**
* Update categories based on ubuntu.com IA: 
  * cloud
  * containers
  * desktop
  * iot
  * packaging - non iot-specific snap/craft content
  * server
* One hidden (not in filter) "Community" category containing the guidelines tutorial
* Server contains the initial Ubuntu on Windows tutorial, assuming it will be used mostly for server-type interactions, as cloud seems to restrictive for it, but this is debatable
* Atom had fun cleaning up trailing spaces, which makes the diff bigger than expected
* One drive-by fix: "Create your own Core Image" -> "Create your own Ubuntu Core image"

### CODE REVIEW

File: tutorials/iot/create-your-own-core-image.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: create-your-own-core-image
 summary: Create your own Ubuntu Core image for a particular model, by assembling snaps available in the store. Make your own device image with some snap preinstalled or additional functionalities!
-categories: snap
+categories: iot
```
Comment (anthonydillon) [inline tutorials/iot/create-your-own-core-image.md]: Should this one be `packaging`?

File: tutorials/iot/setup-ubuntu-core-joule.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 id: setup-ubuntu-core-intel-joule
 summary: In this codelab, we are going to setup Ubuntu Core on your Intel Joule. We’ll flash the sd card with an Ubuntu Core image and then configure it.
-categories: snap
+categories: iot
```
Comment (anthonydillon) [inline tutorials/iot/setup-ubuntu-core-joule.md]: Should this one be `packaging`?

File: tutorials/iot/using-intel-realsense-sdk.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 id: using-intel-realsense-sdk
 summary: We are going to illustrate how to build some Intel RealSense SDK samples on an ubuntu desktop. You will be able to hook up on your Intel Joule and build the samples, explore some of the code and try them!
-categories: snap
+categories: iot
```
Comment (anthonydillon) [inline tutorials/iot/using-intel-realsense-sdk.md]: Should this also be `packaging`? As `iot` has been removed from the filters?

**Comment by WillMoggridge** [comment]
I imagine this isn't effected but Markdown does accept to spaces at the end of a line to insert a new line.

I recommend installing the Editorconfig package in Atom, as there is [.editorconfig](https://github.com/canonical-websites/tutorials.ubuntu.com/blob/master/.editorconfig) file that stops it stripping in Markdown.

The atom package is: https://atom.io/packages/editorconfig

File: tutorials/iot/using-intel-realsense-sdk.md (hunk 3 above)
Comment (caldav) [reply tutorials/iot/using-intel-realsense-sdk.md]: @anthonydillon iot is introduced in this PR, so this change (and the others you pointed out) is accurate, packaging is new as well, but is used for non-iot focused snapping.

**Comment by caldav** [comment]
@WillMoggridge thanks, I wasn't aware of this plugin, I'll add it

File: tutorials/iot/using-intel-realsense-sdk.md (hunk 3 above)
Comment (anthonydillon) [reply tutorials/iot/using-intel-realsense-sdk.md]: Sorry, your right. I miss read the additions.

**Comment by anthonydillon** [comment]
Thanks @caldav :+1:

**Pull request merged**